plugin-packs/postcss-preset-env/.tape.mjs

```diff
@@ -349,6 +329,38 @@ postcssTape(plugin)({
 			}
 		},
 	},
+	'insert:before:match-result:feature-is-skipped': {
+		message: 'supports { insertBefore } usage when looking for a result and the attached feature is skipped',
+		options: {
+			stage: 0,
+			browsers: [
+				'safari >= 15',
+			],
+			insertBefore: {
+				'lab-function': [
+					orderDetectionPlugin('before', (decl) => {
+						return decl.prop === 'color' && decl.value !== 'changed-this-declaration';
+					})
+				]
+			}
+		}
+	},
+	'insert:after:match-result:feature-is-skipped': {
+		message: 'supports { insertAfter } usage when looking for a result and the attached feature is skipped',
+		options: {
+			stage: 0,
+			browsers: [
+				'safari >= 15',
+			],
+			insertAfter: {
+				'lab-function': [
+					orderDetectionPlugin('after', (decl) => {
+						return decl.prop === 'color' && decl.value !== 'changed-this-declaration';
+					})
+				]
+			}
+		}
+	},
```

Comment:
This two test cases had copy/paste issues.
Was intended to test `insertBefore` and `insertAfter` when the attached feature is skipped.

- messages was not acurate
- both had the same config

I also placed them lower in the list.